content/docs/sidebar.yaml

```diff
@@ -18,6 +18,8 @@
       slug: get-started-with-neon/signing-up
     - title: Setting up a project
       slug: get-started-with-neon/setting-up-a-project
+    - title: Regions
```

Comment:
not sure about the placement
i suggest to consider mentioning region in the create a project / set up a project page in the docs
also add regios to glossary?
once that is done not sure if we need a separate getting started guide on this topic